saleor/order/models.py

```diff
@@ -277,7 +277,11 @@ def __str__(self):
         return "#%d" % (self.id,)
 
     def get_last_payment(self):
-        return max(self.payments.all(), default=None, key=attrgetter("pk"))
+        # Skipping a partial payment is a temporary workaround for storing a basic data
+        # about partial payment from Adyen plugin. This is something that will removed
+        # in 3.1 by introducing a partial payments feature.
+        payments = [payment for payment in self.payments.all() if not payment.partial]
```

Comment:
Would be a good option to filter active payments only or do we need all types? If so maybe consider to `filter`?

Reply:
I was thinking about this and also I tried. The issue is action like void, refund etc mark payment as inactive. With the filter we will not return a payment data.

Reply:
Another reason to use `all` instead of filter is that `filter` will not respect prefetches